docs/html-jsx.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML to JSX</title>
+  <!-- Github Pages lacks basic functionality such as permanent server-side
+       redirects, so the best we can do is a meta redirect -->
+  <meta http-equiv="refresh" content="0; url=http://magic.reactjs.net/htmltojsx.htm" />
```

Comment:
I'm going to merge and then follow up to use the `redirect` layout we have set up for this.